SPECS/perl/perl.spec

```diff
@@ -36,19 +40,101 @@
 # same as we provide in /usr/lib/rpm/macros.d/macros.perl
 %global perl5_testdir   %{_libexecdir}/perl5-tests
 
+# Optional features
+# Run C++ tests
+%bcond_without perl_enables_cplusplus_test
+# We can bootstrap without gdbm
+%bcond_without gdbm
+# Support for groff, bug #135101
+%bcond_with perl_enables_groff
+# Run Turkish locale tests
+%bcond_with perl_enables_turkish_test
+# Run syslog tests
+%bcond_with perl_enables_syslog_test
+# SystemTap support
+%bcond_with perl_enables_systemtap
+# <> operator uses File::Glob nowadays. CSH is not needed.
+%bcond_with perl_enables_tcsh
+# We can skip %%check phase
+%bcond_with test
```

Comment:
I think we should replace this with out `%{with_check}`. The way it's set now, it completely skips the contents of the `%check` section regardless which build we're going to run.

Reply:
That's true. However, the current perl %check section is completely empty, and some of the tests are failing. So I didn't want to actually enable the tests yet

Reply:
I see. Could we enable them anyway and say we'll address the test breaks later? With the way things are configured right now there's nothing telling us that the tests are failing, so we'll probably forget to come back to this.